pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -247,7 +499,7 @@ def list_remote_files(tag=None, inst_id=None, start=None, stop=None,
         '1900' will be added for years >= two_digit_year_break
         and '2000' will be added for years < two_digit_year_break.
         (default=None)
-    delimiter : string or NoneType
+    delimiter : str or NoneType
```

Comment:
Strings don't need NoneType as a default unless there's a *really* good reason.

Reply:
I think I got rid of the NoneType default for all the parameters that don't need it. There were a couple with more involved responses to the None value that I left alone. But I'm not 100% sure they couldn't get by with using '' in place of None